### PR TITLE
Use own ping implementation

### PR DESCRIPTION
**Pull request by Nico-Sanchez**
> [!Warning]
> Merge this pr alongside [This client pr](https://github.com/lambdaclass/champions_of_mirra/pull/1975)

## Motivation

Ping was working really weird. It was using cowboy websocket's native implementation which is hard to customize and see what's really going on.
This is part of #794 

## Summary of changes

- [implement new ping and pong messages in game websocket](https://github.com/lambdaclass/mirra_backend/pull/792/commits/d5d201806ba97209634015b06aa61456da545042)
- [Delete unused code](https://github.com/lambdaclass/mirra_backend/pull/792/commits/317f3bd7f032457bc4f36e0f8394bc401a9e74fe)
- [Rename now variable to timestamp_now in proto](https://github.com/lambdaclass/mirra_backend/pull/792/commits/b4c7b523a0e5237c42a421d54c6c0a71bacff0a4)
- [Send ping every 500 ms instead of 30](https://github.com/lambdaclass/mirra_backend/pull/792/commits/b1446e2a1d1be7a7db76d98123372b0ae0b4d260)

## How to test it?

Play a game using client and this branches, you'll see the ping being updated every second precisely.

## Checklist
- [x] Tested the changes locally.
- [x] Reviewed the changes on GitHub, line by line.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.

### CODE REVIEW

**Review by agustinesco — CHANGES_REQUESTED** [review]
The ping logic seems to be correct! but i think there's an error when sending the ping to the clients

File: apps/arena/lib/arena/game_updater.ex [hunk 1]
```diff
@@ -662,6 +692,17 @@ defmodule Arena.GameUpdater do
...
 
+  defp broadcast_ping(state) do
+    now = DateTime.utc_now() |> DateTime.to_unix(:millisecond)
+
+    encoded_state =
+      GameEvent.encode(%GameEvent{
+        event: {:ping, %Ping{timestamp_now: now}}
+      })
+
+    PubSub.broadcast(Arena.PubSub, state.game_id, {:ping, encoded_state})
```
Comment (agustinesco) [inline apps/arena/lib/arena/game_updater.ex:703]: Broadcasting here may be wrong in the ping context since every client will have a different ping

Comment (Nico-Sanchez) [reply apps/arena/lib/arena/game_updater.ex:703]: You're totally right. Nice catch.
Fixed!

**Review by AminArria — APPROVED** [review]
LGTM, just 2 nitpick comments

**Review by tvillegas98 — COMMENTED** [review]
Has my moral approve! but @agustinesco requested changes

**Review by agustinesco — APPROVED** [review]
LGTM! nice job

**Pull request merged**